pydantic/decorator.py

```diff
@@ -100,7 +109,7 @@ def __init__(self, function: 'Callable'):
             # same with kwargs
             fields[self.v_kwargs_name] = Dict[Any, Any], None
 
-        self.create_model(fields, takes_args, takes_kwargs)
+        self.create_model(fields, takes_args, takes_kwargs, **configs)
```

Comment:
better in cases like this to pass the dict, not kwargs kwargs requires a shallow copy of the dict.

Reply:
Maybe we should add a parameter `model` that users can use cumtom model class.

Reply:
Because `extra` should always be `forbid`, so there still exists copy of dict if pass a dict.